scripts/reindex.py

```diff
@@ -167,8 +170,8 @@ def write_lockfile(lockfile, data):
 def verify_collection2_size(data):
     if data['index'].get('numDocs', 0) <= 14150713:
         raise Exception('Too few documents in the new index: %s' % data['index'].get('numDocs', 0))
-    if data['index'].get('sizeInBytes', 0) / (1024*1024*1024.0) <= 146.0: # index size at least 146GB
-        raise Exception('The index is suspiciously small: %s' % (data['index'].get('sizeInBytes', 0) / (1024*1024*1024.0)))
+    if old_div(data['index'].get('sizeInBytes', 0), (1024*1024*1024.0)) <= 146.0: # index size at least 146GB
+        raise Exception('The index is suspiciously small: %s' % (old_div(data['index'].get('sizeInBytes', 0), (1024*1024*1024.0))))
```

Comment:
The default python 3 behavior (or python 2 behavior when `from __future__ import division` is present) is what we want (i.e., 1/2 = 0.5) so we do not need old_div for both lines above.